Source/DafnyServer/boogie/DafnyProvider.cs

```diff
@@ -80,6 +79,29 @@ public DafnyModel(Model m, ViewOptions opts)
       }
     }
 
+    /// <summary>
+    /// Creates a new function that merges together the applications of all the
+    /// functions that have a certain arity and whose name matches the
+    /// "^MapType[0-9]*Select$" pattern. This has previously been done by
+    /// Boogie's Model Parser as a preprocessing step but has been deprecated
+    /// since 2.9.2
+    /// </summary>
+    private Model.Func MergeMapSelectFunctions(int arity) {
+      int id = arity - 1;
+      // Need to make sure the name of the new function is unique
+      while (model.HasFunc("[" + ++id + "]")) { }
+      var result = model.MkFunc("[" + id + "]", arity);
+      foreach (var func in model.Functions) {
+        if (!Regex.IsMatch(func.Name, "^MapType[0-9]*Select$") ||
+            func.Arity != arity) continue;
```

Comment:
Put curly braces around the then branch (here, the `continue;` statement).

Reply:
Done!